sdl_viewer/src/main.rs

```diff
@@ -211,21 +211,27 @@ impl NodeView {
 // Keeps track of the nodes that were requested in-order and loads then one by one on request.
 struct NodeViewContainer {
     node_views: HashMap<octree::NodeId, NodeView>,
-    requested: HashSet<octree::NodeId>,
+    requested: HashSet<octree::NodeId>,     // The node_ids that the I/O thread is currently loading.
+    // Communication with the I/O thread.
     node_id_sender: Sender<octree::NodeId>,
     node_data_receiver: Receiver<(octree::NodeId, octree::NodeData)>,
 }
 
 impl NodeViewContainer {
     fn new(octree: Arc<octree::Octree>) -> Self {
+        // Expensive I/O is performed by an I/O thread. Communication is realized through channels.
```

Comment:
Askew passive tense formulations in comments. They are harder to decipher, reformulate for direct tense: "We perform IO in a separate thread. Data sharing is done through channels."